README.md

```diff
@@ -1 +1,138 @@
-# web-school
\ No newline at end of file
+<a name="readme-top"></a>
+
+
+HOW TO USE:
+Online Website development school
+- Table of Contents
+- About the Project
+  - Built With: HTML, CSS, Javascript
+- Getting Started
+- Authors : Abenezer
+- Future Features
+- Contributing :  Cindy Shin in Behance
+- Show your support
+- Acknowledgements : To Microverse,  Cindy Shin in Behance
+- License MIT.
+
+
+
+
+<!-- TABLE OF CONTENTS -->
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](Html,Css,Javascript)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+- [👥 Authors](Abenezer)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖  Personal Portfolio <a name="about-web-school"></a>
+
+
+
```

Comment:
your readme file should contain links for the following 

- you should make use of GitHub Pages to deploy the web pages.
- The project is deployed properly, and all pages, links, and images work as expected.
- A link to the deployed project is present in the README.